app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3:
```

Comment:
Check that the "cp" input command is correct.

Reply:
Fix that. You have this point in your checklist, don't forget to read it before creating a pull request